_tools/check_docs.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * This script checks that all exported functions have JSDoc comments with
+ * `@param`, `@return`, and `@example` tags, according to the contributing
+ * guidelines.
+ *
+ * @see {@link https://github.com/denoland/deno_std/blob/main/.github/CONTRIBUTING.md#documentation}
+ *
+ * @todo(iuioiua) Add support for classes and methods.
```

Comment:
Can you align the TODO comment format to the style guide? https://docs.deno.com/runtime/manual/references/contributing/style_guide#todo-comments

Reply:
Done